src/ContentManager.php

```diff
@@ -136,6 +140,36 @@ public function getContent(string $type, string $id): object
         throw new \RuntimeException(sprintf('Content not found for type "%s" and id "%s".', $type, $id));
     }
 
+    public function reverseContent(array $context): ?Content
```

Comment:
🤔

Reply:
Just to explicit the use-cases a bit, and give a frame to it, I've added `Context` and `RelativeLinkContext` classes instead of this array.
One can implement their own contexts according to their needs and handle in their custom providers.